ComprefaceTestApp/DTOs/ExampleSubject/DeleteMultipleExamples/DeleteMultipleExamplesRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ComprefaceTestApp.DTOs.ExampleSubject.DeleteImageById;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ComprefaceTestApp.DTOs.ExampleSubject.DeleteMultipleExamples
+{
+    public class DeleteMultipleExamplesRequest
+    {
+        public IList<Guid>? ImageIdList { get; set; }
```

Comment:
IList is also by default nullable so no need for `?` here too

Reply:
From C# 8.0 null-context definition has appeared. So that's why we need mark string like a nullable. Also we can set up it global in the project and disable this warnings.